src/constbuffer.c

```diff
@@ -98,7 +98,9 @@ CONSTBUFFER_HANDLE CONSTBUFFER_CreateFromBuffer(BUFFER_HANDLE buffer)
     }
     else
     {
-        result = (CONSTBUFFER_HANDLE_DATA*)CONSTBUFFER_Create_Internal(BUFFER_u_char(buffer), BUFFER_length(buffer));
+        size_t length = BUFFER_length(buffer);
+        unsigned char* rawBuffer = BUFFER_u_char(buffer);
+        result = (CONSTBUFFER_HANDLE_DATA*)CONSTBUFFER_Create_Internal(rawBuffer, length);
```

Comment:
I know. I suggest asking the author for a new feature .

---
In reply to: [75430488](https://github.com/Azure/azure-c-shared-utility/pull/21#discussion_r75430488) [](ancestors = 75430488)